config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users, path: '', path_names: { sign_in: 'login', sign_out: 'logout', sign_up: 'register'}
```

Comment:
`devise_for :users` should work as expected

Reply:
i.e. i need to remove what goes after users?

Reply:
yes, it should work without it

Reply:
also remove next lines with `devise_scope`

Reply:
Removed.